docs/how-to-guides/curating-the-editor-experience.md

```diff
@@ -0,0 +1,314 @@
+```
+{
+	"version": 2,
+	"settings": {
+		"color": {
+			"duotone": [
+				{
+					"colors": [ "#000000", "#ffffff" ],
+					"slug": "foreground-and-background",
+					"name": "Foreground and background"
+				},
+				{
+					"colors": [ "#000000", "#ff0200" ],
+					"slug": "foreground-and-secondary",
+					"name": "Foreground and secondary"
+				},
+				{
+					"colors": [ "#000000", "#7f5dee" ],
+					"slug": "foreground-and-tertiary",
+					"name": "Foreground and tertiary"
+				},
+			]
+		}
+	}
+}
+```
+
+*Duotone with defined default options and all customization available for the Post Featured Image block:*
+
+```
+{
+	"schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 2,
+	"settings": {
+	      "color": {
+              	"custom": true,
+              	"customDuotone": true
+        	},
+		"blocks": {
+              	"core/post-featured-image": {
+                   		"color": {
+                          		"duotone": [
+						{
+                   		 			"colors": [ "#282828", "#ff5837" ],
+                   		 			"slug": "black-and-orange",
+                   		 			"name": "Black and Orange"
+               				},
+						{
+                   		 			"colors": [ "#282828", "#0288d1" ],
+                   		 			"slug": "black-and-blue", 
+                   		 			"name": "Black and Blue"
+               				}
+					],
+                          		"customDuotone": true,
+                          		"custom": true
+				}
+			}
+} 
+}
+}
+```
+
+*Duotone with only defined default options and core options available for the Post Featured Image block (no customization):*
+
+```{
+	"schema": "https://schemas.wp.org/trunk/theme.json",
+	"version": 2,
+	"settings": {
+	      "color": {
+              	"custom": true,
+              	"customDuotone": true
+},	
+		"blocks": {
+              	"core/post-featured-image": {
+                   		"color": {
+                          		"duotone": [
+						{
+                   		 			"colors": [ "#282828", "#ff5837" ],
+                   		 			"slug": "black-and-orange",
+                   		 			"name": "Black and Orange"
+               				},
+						{
+                   		 			"colors": [ "#282828", "#0288d1" ],
+                   		 			"slug": "black-and-blue",
+                   		 			"name": "Black and Blue"
+               				}
+					],
+                          		"customDuotone": false,
+                          		"custom": false
+				}
+			}
+} 
+}
+}
+```
+
+## Limiting interface options with theme.json
+
+**Limit options on a per block basis**
+
+Beyond defining default values, using theme.json allows you to also remove options entirely and instead rely on what the theme has set in place. Below is a visual showing two extremes with the same paragraph block: 
+
+![Image of restricted interface](https://github.com/WordPress/gutenberg/blob/annezazu-curate-new-doc/docs/assets/Locking%20comparison%20visual.png)
```

Comment:
I think the URL of the image already needs to point to the path of the image once it has been merged to trunk.

Reply:
Updated but, as you can see now, the images don't show up. I guess that's what has to be done but I'm not sure what folks have done previously.